### DISCUSSION TITLE
TIile images using libvips

### DISCUSSION THREAD

**Discussion by AdigaAkhil** [libvips/libvips#3095] [Q&A] [answered]
I have been tiling images till now using GDAL and recently I stumbled upon libvips library. I searched through the website on tiling images but I was not able to Assign the georeferenced bounds for the tiles generated from the image. These are the below commands which I used in GDAL. It would be helpful if the same was achieved in libvips.

**To generate VRT file with coordinates:**

`gdal_translate -of  GTiff -a_srs EPSG:4326 -a_ullr 12.88375724563124 74.83986362864423 12.883615511245678 74.840137447214213 foo.png foo.tif`

**To generate tiles:**

`gdal2tiles.py --processes=2 foo.tif`

**Comment by jcupitt** [comment, marked as answer]
Hello @AdigaAkhil,

I think this is out of scope for libvips. It does not support the extra gdal TIFF tags, and it'd be a lot of work to add this feature.

**Reply by AdigaAkhil** [reply]
Ohh, what about assigning georeferenced bounds to tiles? Also what about VRT tags?

**Reply by jcupitt** [reply]
We'd need to add code for handling the special GDAL TIFF tags to libvips for that.

**Reply by AdigaAkhil** [reply]
ok thanks
